src/routes.jsx

```diff
@@ -0,0 +1,26 @@
+import { parse } from 'query-string';
+import { Redirect, Route, Switch } from 'react-router-dom';
+
+import Benchmark from './views/Benchmark';
+
+const Routes = () => (
+  <Switch>
+    <Route
+      path="/"
+      render={({ location }) => {
+        const { platform, benchmark } = parse(location.search);
+        if (!platform || !benchmark) {
+          return <Redirect to="/?platform=win10&benchmark=overview" />;
```

Comment:
Curious as to why this needs to be forced into the URL vs. just having default variables here, avoiding the redirect:

```jsx
<Route path="/" component={Benchmark} />

// in Benchmark component:
class Benchmark extends Component {
  render() {
    const {
      platform = 'win10',
      benchmark = 'overview'
    } = parse(this.props.location.search);
```